commonutils/src/main/java/io/intrepid/commonutils/IntrepidJavaUtils.java

```diff
@@ -0,0 +1,56 @@
+package io.intrepid.commonutils;
+
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import java.util.Collections;
+import java.util.List;
+
+public class IntrepidJavaUtils {
+
+    @NonNull
+    public static String bytesToHexString(@NonNull byte[] bytes) {
```

Comment:
I tried using the BigInteger version, but its result doesn't include the leading 0.